scipy/stats/_levy_stable/__init__.py

```diff
@@ -531,6 +532,11 @@ def _fitstart_S1(data):
         [0, 0.064, 0.128, 0.191, 0.330, 0.478, 1.362],
         [0, 0.056, 0.112, 0.167, 0.285, 0.428, 1.274]]
 
+    j = np.argsort(nu_alpha_range)
```

Comment:
Is this just to emphasize that the data has to be in order? It seems unnecessary, because the current tables _are_ in order, but I would understand if the intent is to future-proof the code.
(Since I'm not very familiar with `RectBivariateSpline` or `interp2d`, I am not a bit less confident about my understanding.)

Reply:
Yes. This stanza is straight from interp2d code. RectBivariateSpline errors out if the data is not sorted in the right order.

Reply:
Sorry, should not type before coffee. Let me try again: RectBivariateSpline requires the input in the ascending order, and errors otherwise. 
interp2d however argsorts the inputs. 
This all would unnecessary if the tables were in the ascending order, but they are not and I felt it's safer to add some uglyness then redo the tables.

Reply:
Well, the first tables are already in order. They only need to be transposed. I wouldn't mind just adding `np.array` around the definition of the table itself and appending `.T` at the end. Then we wouldn't need the `.T` when you pass it into `RectBivariateSpline`. 

And the set of tables are based on `alpha_range` in reverse order, so the rows just need to be reversed before the transpose. Similarly, I wouldn't mind doing all that in the definition by packing the list into an array, reversing the rows and transposing `[::-1].T`.

I just think it would make the logical part of the code a little simpler if the tables were set up the way they are going to be used, and we can do that pretty simply without a full `argsort`. Adding a comment e.g. "Order with increasing `alpha_range` and transpose as required by `RectBivariateSpline`" might help clarify for future readers.

I tried it locally. Let me know if you'd like me to suggest it as a PR to your branch.

Reply:
Yes, I was lazy and simply copied what interp2d was doing, with argsorts and transposes and all.  Please feel free to refactor however you deem better!